README.md

```diff
@@ -176,12 +176,16 @@ We're considering:
 
 ### Building a new package
 
+To build Data Curator for macOS locally:
+
+`yarn build`
```

Comment:
`yarn build` needs something else as it losses toolsbar and about images in the build process

Reply:
@Stephen-Gates 
Should be `yarn run build` - no images should be lost now with path changes (let me know if still not the case)
Yes just need to bump version number in package.json - about panel will read from this.

Reply:
Yep that worked fine @mattRedBox